<?xml version="1.0" encoding="UTF-8"?>
<testrun name="MultiDataJUnitRunnerTest" project="Test" tests="2" started="2" failures="1" errors="0" ignored="0">
  <testsuite name="org.dejava.component.util.test.test.MultiDataJUnitRunnerTest" time="209.077">
    <testcase name="testTestNormalSuccess" classname="org.dejava.component.util.test.test.MultiDataJUnitRunnerTest" time="0.159"/>
    <testcase name="testTestNormalAssumptionFailure" classname="org.dejava.component.util.test.test.MultiDataJUnitRunnerTest" time="208.918">
      <failure>junit.framework.AssertionFailedError: null&#13;
	at junit.framework.Assert.fail(Assert.java:47)&#13;
	at junit.framework.Assert.fail(Assert.java:53)&#13;
	at org.dejava.component.util.test.test.MultiDataJUnitRunnerTest.testTestNormalAssumptionFailure(MultiDataJUnitRunnerTest.java:75)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:168)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
  </testsuite>
</testrun>
